src/routes/index.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  res.redirect('/qa');
```

Comment:
this is dope, I haven't used this.

Reply:
Yeah, this was kinda weird. I guess it was needed for my routes since everything I serve is under `qa` and if someone ever stumbles on my root, it'll just forward the calls to `qa`.